cmd/create.go

```diff
@@ -93,22 +93,30 @@ A full list of component types that can be deployed is available using: 'odo com
 			os.Exit(1)
 		}
 
+		fmt.Printf("Creating component %s from build type %s\n", componentName, componentType)
+
 		if len(componentGit) != 0 {
+			fmt.Printf("Building component %s from Git repository: %s\n\n", componentName, componentGit)
+
 			err := component.CreateFromGit(client, componentName, componentType, componentGit, applicationName)
 			checkError(err, "")
-			fmt.Printf("Component '%s' was created.\n", componentName)
-			fmt.Printf("Triggering build from %s.\n\n", componentGit)
 			err = component.RebuildGit(client, componentName)
 			checkError(err, "")
+
 		} else if len(componentLocal) != 0 {
+
 			// we want to use and save absolute path for component
 			dir, err := filepath.Abs(componentLocal)
 			checkError(err, "")
+
+			fmt.Printf("Building component %s from local directory %s", componentName, dir)
```

Comment:
missing `\n` 
there should be proabbly `\n\n` to make it same as in `--git` component

Reply:
Wouldn't it be better without world "component"? Don't know, just something to think about ;-)